### ISSUE TITLE
Performance drop

### ISSUE THREAD

**Issue by SelfDestruct68** [subliun/Antox#141]
Hi developers, I got quite an annoying bug here. The longer antox runs the more the performance drops. De app becomes very slow and typing is almost impossible. After I force close the app an restart it, everything will be fine. And after a while it begins to drop performance again.

Android 5.0.1
Antox 0.23.41

**Comment by LuccoJ** [comment]
For me, more than stopping the app and restarting it, it seems to be about someone's conversation logs getting longer. When I tell Antox to wipe the log, it becomes fast again.

**Comment by agilob** [comment]
Clean chat history...

**Comment by LuccoJ** [comment]
Yes, that's what I just said. That doesn't mean it should be the case: a little chatting (not days, weeks or months) causes the chat history to become too long already for fast operation. If anything, the client should prune the chat history automatically (but ideally it would just not slow down things so much).
